Bottom: 5d3091e01bf2019b7f766a1ce4790fe95658ba41
Top:    228b5569890f9a65c59d74f5545692df5757fecf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-02 00:38:57 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index a522c96..d3eda39 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
